e2e/definitions/secrets/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+- secrets.yaml
```

Comment:
there is no namespace for the secrets? If not, the `default` namespace is being used. I propose to make this explicit